### PR TITLE
{bio}[intel/2019.02] ROME v1.1.2

### PR DESCRIPTION
**Pull request by timeu**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/ROME/ROME-1.1.2_bool-nullptr.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- ROME1.1.2/src/map3d_optimizer.cpp.orig	2019-09-27 00:08:10.616171000 +0200
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/r/ROME/ROME-1.1.2_bool-nullptr.patch]: please add description and author here

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/973cf2ad693d78214afdfd0269854646 for a full test report.

File: easybuild/easyconfigs/r/ROME/ROME-1.1.2-intel-2019.02.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+files_to_copy = ['bin']
+
+local_files = ['rome_deep2d', 'rome_map2d', 'rome_map3d', 'rome_reconstruct', 'rome_res', 'rome_sml', 'rome_tool']
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in local_files],
+    'dirs': [],
+}
+
+
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/r/ROME/ROME-1.1.2-intel-2019.02.eb]: style nitpicking: remove extra line?

**Comment by migueldiascosta** [comment]
Going in, thanks @timeu!

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2616.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/67af94f26d9a61149181a7dfa7fd6bc6 for a full test report.

**Comment by boegel** [comment]
@timeu @migueldiascosta This is broken, checksums for patch wasn't updated after the description was added to the patch...

**Comment by migueldiascosta** [comment]
oops - fix in #9062

**Comment by timeu** [comment]
@boegel , @migueldiascosta  sorry for the mistake

**Comment by boegel** [comment]
@timeu Don't worry about it, this is why we have a `develop` branch. :)

And the tests could actually verify the checksums for patches for easyconfig files touched in a PR, we should look into that...

**Pull request merged**